WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -804,6 +806,8 @@ - (void)showSettings
     PostSettingsViewController *vc = [[[self classForSettingsViewController] alloc] initWithPost:post];
 	vc.hidesBottomBarWhenPushed = YES;
     [self.navigationController pushViewController:vc animated:YES];
+
+    [self.post addObserver:self forKeyPath:@"dateCreated" options:NSKeyValueObservingOptionNew context:DateChangeObserverContext];
```

Comment:
Technically the field in question is `date_created_gmt` but the accessor methods in BasePost.m should work for `setDateCreated:`. I don't see `date_created_gmt` being used anywhere outside of sync and a couple sort descriptors.